AEPIdentity/Tests/IdentityFunctionalTests.swift

```diff
@@ -418,4 +430,8 @@ class IdentityFunctionalTests: XCTestCase {
         let ecid = dispatchedEvent?.data?[IdentityConstants.EventDataKeys.VISITOR_ID_ECID] as? String ?? ""
         XCTAssertFalse(ecid.isEmpty)
     }
+
+    func mockLastSyncTimeStamp(ts: Date = Date()) {
```

Comment:
can we also remove mockLastSyncTimeStamp and waitForProcessing as they are not used?

Reply:
oh yes. Thanks for catching it!